_templates/footer-extra.html

```diff
@@ -0,0 +1,26 @@
+<div class="container-fluid footer-extra">
+  <div class="container">
+    <center>
+      Theme by <a href="https://projectpythia.org">Project Pythia</a>. All code
+      in Pythia Foundations is licensed under Apache 2.0. All other non-code
```

Comment:
```suggestion
      in Pythia Cookbooks is licensed under [Apache 2.0](https://opensource.org/license/apache-2-0). All other non-code
```

Reply:
Oops, actually the markdown link syntax won't work here.

Reply:
Swapped that for HTML link

Reply:
Thanks should have checked for that